Test Policy

```diff
@@ -0,0 +1,65 @@
+Purpose and Scope 
```

Comment:
This document needs to be under docs/quality-assurance... assume it is possible to move within GitHub!